package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@curefit/react-native-video",
-    "version": "5.1.5",
+    "version": "5.1.6",
```

Comment:
5.1.7 will have conflict as was previously used - so might get npm publish conflicts - please jump this to 5.2 series in next publish